cmake/googletest/CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ if(BUILD_TEST)
             WORKING_DIRECTORY "${SOURCE_DIR}"
             RESULT_VARIABLE result)
         if(result)
-            message(FATAL_ERROR "CMake step [patch] for googletest failed: ${result}")
+            message(WARNING "CMake step [patch] for googletest failed: ${result}! Build of gtest might fail")
```

Comment:
had to do this since I had problems running cmake mutliple times

Reply:
Mmmhhh thats not so nice. Is it possible to run this command only once when we clone the repo and when its already present we skip it?

Reply:
I could not find a solution for this but in the end it doesn't matter. If this doesn't stop the build the `-Werror` does